install-usercss/install-usercss.js

```diff
@@ -0,0 +1,291 @@
+/* global CodeMirror semverCompare makeLink closeCurrentTab runtimeSend */
+'use strict';
+
+(() => {
+  const params = new URLSearchParams(location.search);
+  let liveReload = false;
+  let installed = false;
+
+  const port = chrome.tabs.connect(
+    Number(params.get('tabId')),
+    {name: 'usercss-install', frameId: 0}
+  );
+  port.postMessage({method: 'getSourceCode'});
+  port.onMessage.addListener(msg => {
+    switch (msg.method) {
+      case 'getSourceCodeResponse':
+        if (msg.error) {
+          alert(msg.error);
```

Comment:
prefer messageBox() over JS dialogs!!!!

Reply:
Should `confirm()` also be replaced with `messageBox()`? They are used in `$('button.install').onclick` handler.

Reply:
NVM I saw the comment below.